### ISSUE TITLE
Investigate difference between case lists and genomic profile sample counts

### ISSUE THREAD

**Issue by callachennault** [knowledgesystems/pipelines-scrum#1274]
## Done Condition (What do we need? Why do we need it? Keep this is small as possible!)
- [x] Understand difference between case lists and genomic profile sample counts
- [x] Confirm SV case lists logic with curators
- [x] Investigate difference between sequenced samples and gene matrix+MAF files - requeue samples missing data
- [x] Document case list / genomic profile sample count logic in Confluence
- [x] Reply to Ambar's email explaining discrepancy / whether any changes were made

## Technical Description (How are we going to achieve the above)
Ambar from AstraZeneca reached out about a discrepancy in the MSK-IMPACT data. There is a difference between the numbers in the Genomic Profile Sample Counts chart and the Case Lists chart for CNA data and SV data (examples below).
<img width="903" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBelpoQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0b112f85be233ecd11ec1d76972947b0dd197399/image.png" alt="image.png" />
<img width="928" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBemRoQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--03935c4b633de1fbbad301b646a8a0f810363be3/image.png" alt="image.png" />

**Comment by callachennault** [comment]
Case lists: Samples representing cases with actual events within the portal
Genomic profile sample counts: Any sample that our system has designated as having a profile or analysis run on it. There is no guarantee that a sample that was profiled will have an event.

The discrepancy between CNA case lists (106,519) and CNA genomic profile sample count (106,529) is consistent with the logic above. There are more profiled samples than samples with events.
The discrepancy between SV case lists (107,110) and SV genomic profile count (106,529) is slightly different. There is extra logic in the case lists script that uses the sequenced_samples heading in the MAF file as the case list for SV data. Meaning, this number represents all mutations within the portal - not just SV mutations. TODO: confirm this logic with curation.

When looking into the SV case lists discrepancy, we also noticed that there is a difference between the number of samples in the sequenced_samples header and the actual samples present in the MAF file + data gene matrix file. We diffed these files and requeued missing samples on 4/15
